rapier/tests/acceleration.rs

```diff
@@ -0,0 +1,124 @@
+#![cfg(all(
+    any(feature = "2d", feature = "3d"),
+    not(all(feature = "2d", feature = "3d")),
+))]
+
+use bevy::prelude::{GlobalTransform, Transform};
+use bevy::reflect::TypeRegistryArc;
+use bevy_app::App;
+use bevy_core::CorePlugin;
+use bevy_math::Vec3;
+use heron_core::{Acceleration, AxisAngle, Body};
+use heron_rapier::convert::IntoBevy;
+#[cfg(feature = "3d")]
+use heron_rapier::rapier::math::Vector;
+use heron_rapier::{
+    rapier::dynamics::{IntegrationParameters, RigidBodySet},
+    BodyHandle, RapierPlugin,
+};
+
+fn test_app() -> App {
+    let mut builder = App::build();
+    builder
+        .init_resource::<TypeRegistryArc>()
+        .add_plugin(CorePlugin)
+        .add_plugin(RapierPlugin {
+            step_per_second: None,
+            parameters: {
+                let mut params = IntegrationParameters::default();
+                params.dt = 1.0;
+                params
+            },
+        });
+    builder.app
+}
+
+#[test]
+fn body_is_created_with_acceleration() {
+    let mut app = test_app();
+
+    #[cfg(feature = "3d")]
+    let linear = Vec3::new(1.0, 2.0, 3.0);
+    #[cfg(feature = "2d")]
+    let linear = Vec3::new(1.0, 2.0, 0.0);
+
+    let angular = AxisAngle::new(Vec3::unit_y(), 1.0);
```

Comment:
Here you rotate around the y axis, but I think in 2d space you should rotate around the z axis like you do in the other test. If I use unit_z here the test passes for 2d as well as 3d, except that in 2d there is the small constant deviance.

We could use the `assert_float_eq` to deal with those minor differences if we decide it doesn't hint at an underlying problem.

Reply:
Yep, that was actually why the test was failing. I fixed in now^^

I also updated `assert_eq_angular` to accept floating points errors.